### PR TITLE
Kangz/misc

### PR DESCRIPTION
**Pull request by Kangz**
PTAL, stuff found while working on the asserts and logs branch. Thanks!

### CODE REVIEW

File: daemon/src/common/Color.cpp [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /*
 ===========================================================================
 Daemon BSD Source Code
-Copyright (c) 2015, Daemon Developers
+Copyright (c) 2013-2016, Daemon Developers
 All rights reserved.
```
Comment (mbasaglia) [inline daemon/src/common/Color.cpp]: Is this attribution correct? This code got introduced in 2015

File: daemon/src/common/Platform.h [hunk 2]
```diff
@@ -32,7 +32,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #define COMMON_PLATFORM_H_
 
 // Platform-specific configuration
-#ifdef _WIN32
+#if defined(_WIN32)
```
Comment (mbasaglia) [inline daemon/src/common/Platform.h]: Any particular reason for this style change?

File: src/sgame/sg_api.cpp [hunk 3]
```diff
@@ -1,22 +1,30 @@
...
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL DAEMON DEVELOPERS BE LIABLE FOR ANY
+DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```
Comment (mbasaglia) [inline src/sgame/sg_api.cpp]: hmm... will this cause licensing issues?

File: daemon/src/common/Color.cpp (hunk 1 above)
Comment (Kangz) [inline daemon/src/common/Color.cpp]: I think it is valid to say that the "Daemon BSD source code" (which this file is part of) started in 2013 and it makes it easier to update all the copyrights in the future.

File: daemon/src/common/Platform.h (hunk 2 above)
Comment (Kangz) [inline daemon/src/common/Platform.h]: I like it better than ifdef but it is just a change I made by reflex, can be removed if you want.

File: src/sgame/sg_api.cpp (hunk 3 above)
Comment (Kangz) [inline src/sgame/sg_api.cpp]: No, we wrote this file from scratch (function declarations may have been copied but it is unavoidable to make the functions declarations look the same)

Comment (Kangz) [inline src/sgame/sg_api.cpp]: This file was written a long time ago when we started porting sgame to NaCl and is a reimplementation of parts of the old QVM handling code in g_main.c and g_api.c to use NaCl instead.

File: daemon/src/common/Platform.h (hunk 2 above)
Comment (DolceTriade) [inline daemon/src/common/Platform.h]: I think it's fine. The rest of the code (except the include guard) uses #if defined

**Comment by DolceTriade** [comment]
LGTM

**Comment by Kangz** [comment]
Thanks, merging.

**Pull request merged**